bot/exts/fun/uwu.py

```diff
@@ -0,0 +1,144 @@
+import random
+import re
+from functools import partial
+from typing import Callable
+
+from discord.ext import commands
+from discord.ext.commands import Cog, Context, clean_content
+
+from bot.bot import Bot
+from bot.exts.fun.fun import Fun
+from bot.utils import helpers
+
+
+WORD_REPLACE_HASH = {
+    "small": "smol",
+    "cute": "kawaii~",
+    "fluff": "floof",
+    "love": "luv",
+    "stupid": "baka",
+    "idiot": "baka",
+    "what": "nani",
+    "meow": "nya~",
+    "roar": "rawrr~",
+}
+
+EMOJI_LUT = [
+    "rawr x3",
+    "OwO",
+    "UwU",
+    "o.O",
+    "-.-",
+    ">w<",
+    "σωσ",
+    "òωó",
+    "ʘwʘ",
+    ":3",
+    "XD",
+    "nyaa~~",
+    "mya",
+    ">_<",
+    "rawr",
+    "uwu",
+    "^^",
+    "^^;;",
+]
+
+word_replace_regex = re.compile(r"(?<![w])[lr](?![w])")
+
+punctuation_regex = re.compile(r"[.!?\r\n\t]")
+
+stutter_regex = re.compile(r"(\s)([a-zA-Z])")
+stutter_substitute = "\\g<1>\\g<2>-\\g<2>"
+
+nya_regex = re.compile(r"n([aeou])([^aeiou])")
+nya_substitute = "ny\\g<1>\\g<2>"
```

Comment:
The `r''` actually stands for raw, it is mainly useful with backslashes. `r'\n'` becomes the actual string `'\n'` while `'\n'` in the code becomes a new-line character. It is pretty common to think `r` is for regex but this is not correct.

The reason I say this is because you double-backslash in the subtitute strings, and always use raw strings in the regex compilation.

*Side note: these should be all-caps for CONSTANTS*.

Reply:
That's good information, will update when I can.

Reply:
Done.